BendAssist.App/BendAssists/BendRelief.cs

```diff
@@ -1,24 +1,81 @@
 ﻿using BendAssist.App.Model;
+using BendAssist.App.Utils;
 
 namespace BendAssist.App.BendAssists;
 
 #region class BendRelief --------------------------------------------------------------------------
 public sealed class BendRelief : BendAssist {
-   #region Constructors ---------------------------------------------
+   #region Constructors --------------------------------------------
    public BendRelief (Part part) => mPart = part;
-   #endregion
+   #endregion
 
-   #region Methods --------------------------------------------------
+   #region Methods -------------------------------------------------
+   /// <summary>Method to assess whether bend relief necessary for slected base and bend line</summary>
    public override bool Assisted () {
+      return true;
+   }
 
+   /// <summary>Applies bend relief for the given part</summary>
+   public override void Execute () {
+      if (mPart is null || mPart.BendLines is null || mPart.BendLines.Count is 0) return;
+      mCommonVertices = [..mPart.Vertices.GroupBy (p => p).Where (p => p.Count () == 2).Select (g => g.Key)
+                       .Where (x => x.IsWithinBound (mPart.Bound))];
+      if (mCommonVertices is null) return;
+      List<PLine> pLines = [.. mPart.PLines];
+      foreach (var vertex in mCommonVertices) {
+         foreach (var bl in mPart.BendLines) {
+            if (bl.HasVertex (vertex) && bl.Orientation is not EOrientation.Inclined) {
+               List<PLine>? connectedLines = [.. pLines.Where (x => x.HasVertex (vertex)).Where (x => x.Orientation == bl.Orientation)];
+               if (connectedLines.Count is 0) return;
+               PLine nearBaseEdge = connectedLines.First (); //Get the near base edge where relief edges will be inserted
+               bool isHorizontal = bl.Orientation == EOrientation.Horizontal;
+               Point2 p1 = vertex, p2, p3, p4, center = mPart.Centroid;
+               (float blAngle, float radius, float deduction) = bl.BLInfo;
+               //calculating the benrelief height from bend allownace or flat width

```

Comment:
Please check  the typo and turn on the spell check in the development environment